Cql/Cql.Comparers/CqlComparers.cs

```diff
@@ -239,14 +232,11 @@ public CqlComparers Unregister(Type type)
                     comparer = enumerableComparer;
                 }
             }
-            else if (x is TupleBaseType && Comparers.TryGetValue(typeof(TupleBaseType), out ICqlComparer? tupleComparer))
```

Comment:
Why is checking for TupleBaseType not necessary anymore? Will Comparers.TryGetValue() now return it? Or did it actually always already do that?

Reply:
The `GetType` method handles this above. And this is the `GetType` method:

```csharp
 private static Type GetType(object? x)
 {
     var type = x switch
     {
         TupleBaseType => typeof(TupleBaseType),
         ITuple        => typeof(ITuple),
         _             => x!.GetType()
     };
     return type;
 }
```